chef-config/lib/chef-config/version.rb

```diff
@@ -20,6 +20,6 @@
 #!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
 
 module ChefConfig
-  VERSION = '12.5.0.current.0'
+  VERSION = '12.5.0.alpha.0'
```

Comment:
Yeah, Claire's reasoning is why we don't do that. `chef-config` is its own gem, it's just in the same repo as chef (the same way rails is). So we have a `VERSION` file at the root and a rake task to do the update everywhere.